doc/sources/.templates/layout.html

```diff
@@ -177,6 +185,14 @@ <h3>Related Topics</h3>
                   <li><a class="nav-api" href="{{ pathto('api-kivy') }}">API Reference</a></li>
                   <li><a class="nav-pdf" href="https://media.readthedocs.org/pdf/kivy/latest/kivy.pdf">PDF</a></li>
                   <li><a class="nav-wiki" href="http://wiki.kivy.org">Wiki</a></li>
+                  <li>
+                      <div id="themeToggleSwitchContainer">
```

Comment:
Need some ideas for how the switch should look 
currently 
![image](https://user-images.githubusercontent.com/22869882/153664301-59f74654-f240-46bf-8c4b-1bc19f38f6d8.png)

Reply:
Just thinking out loud:

- Instead of placing it vertically,  can be placed horizontally with a couple of icons? (sun and moon)
- Desktop sizes: placed at the left of the wiki button
- Smartphone sizes: Into this area?
![Image 11-02-22 at 22 15](https://user-images.githubusercontent.com/8177736/153671395-0ef8c0ac-bf54-4f33-89ab-4de3912d84e3.jpg)